src/main/java/edu/kis/powp/jobs2d/drivers/figureFactory.java

```diff
@@ -0,0 +1,62 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.commands.ComplexCommand;
+import edu.kis.powp.jobs2d.commands.DriverCommand;
+import edu.kis.powp.jobs2d.commands.OperateToCommand;
+import edu.kis.powp.jobs2d.commands.SetPositionCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.Math.cos;
+import static java.lang.Math.sin;
+
+
+
+public class figureFactory {
```

Comment:
A class name should start from a lowercase.
Neither of the factory methods is tested!!!